corehq/apps/reports/standard/cases/data_sources.py

```diff
@@ -206,7 +206,7 @@ def server_last_modified_date(self):
     def owner_display(self):
         owner_type, owner = self.owner
         if owner_type == 'group':
```

Comment:
just wondering if we should apply the same formatting to locations?

Reply:
locations seems to be doing okay, https://www.commcarehq.org/a/goldsmith-foundation/reports/case_list/?case_list_filter=project_data&case_type=&is_open=&search_query= ?

Reply:
yes, they are fine - just wondering why groups get special attention